### PR TITLE
Re-enables `--k8s-users` & `--k8s-groups` in tctl users add

### PR DESCRIPTION
**Pull request by tcsc**
See-Also: Issue #6342

### CODE REVIEW

File: tool/tctl/common/user_command.go [hunk 1]
```diff
@@ -46,8 +46,8 @@ type UserCommand struct {
 	// DELETE IN (7.0)
 	// We keep legacy flags used in tctl users add --k8s-users command
 	legacyAllowedLogins string
-	legacyKubeUsers     string
-	legacyKubeGroups    string
+	kubeUsers           string
+	kubeGroups          string
```
Comment (nklaassen) [inline tool/tctl/common/user_command.go:50]: nit: maybe separate these from the `DELETE IN (7.0)` comment. Not sure what the comment is saying about `--k8s-users` either

**Review by nklaassen — APPROVED** [review]
LGTM

**Review by r0mant — COMMENTED** [review]
Changes look good if @klizhentas is onboard with them (he removed these settings when working on OSS RBAC).

Also while we're here - would it make sense to bring back `--db-users` and `--db-names` in a similar fashion here too then? It is also not very straightforward to set these traits currently - have to do it using resource-based approach.

**Comment by russjones** [comment]
@r0mant Yeah, @klizhentas is onboard, see: https://github.com/gravitational/teleport/issues/6342.

**Review by r0mant — APPROVED** [review]
lgtm if we don't want to bring back db names/users as a part of this.

**Comment by tcsc** [comment]
Will add `--db-*` flags in a later PR

**Pull request merged**